apps/web-giddh/src/app/shared/header/header.component.html

```diff
@@ -838,11 +840,11 @@ <h6>Downloads</h6>
 </ng-template>
 
 <!--help & support sidebar  -->
-<div class="help-support-aside" *ngIf="asideHelpSupportMenuState === 'in'">
+<div class="help-support-aside" *ngIf="asideHelpSupportMenuState === 'in'" (clickOutside)="closeHelpPanOnOutsideClick()">
     <aside-help-support (closeAsideEvent)="toggleHelpSupportPane(false)"></aside-help-support>
 </div>
 <!-- setting aside pane -->
-<div class="help-support-aside" *ngIf="asideSettingMenuState === 'in'">
+<div class="help-support-aside" *ngIf="asideSettingMenuState === 'in'" (clickOutside)="closeSettingPanOnOutsideClick()">
```

Comment:
Avoid spelling mistakes
```suggestion
<div class="help-support-aside" *ngIf="asideSettingMenuState === 'in'" (clickOutside)="closeSettingPaneOnOutsideClick()">
```

Reply:
done